### ISSUE TITLE
[1.12][Enhancement][Mod Compatibility] Better Animal Models Spider Model Compatibility

### ISSUE THREAD

**Issue by SandwichHorror** [TheCyberBrick/Spiders-2.0#11]
[Better Animal Models](curseforge.com/minecraft/mc-mods/better-animal-models) adds upgraded/more realistic spider models.

Similar to Mo'Bends before you added the special compatibility patch, these custom models also have "partial rotation" when climbing walls, and only apply to the vanilla spider and cave spider models. Some care would have to be taken to make them apply to the new entities and not act odd when rotating.

If you are able, it would be great to see compatibility made possible for these creepy arachnids for those who prefer these models instead of Mo'Bends.

Thanks!

**Comment by TheCyberBrick** [comment]
Better Animal Models seems to already have things set up for mod integrations from their side, so it might be best to add the integration to their mod instead. I might make a pull request though.

**Comment by Jaeric217** [comment]
I would love to have support between these two mods too, but I don't think Better Animal Models is doing any more updates for 1.12.2 so it's unlikely to happen from their side.

Edit: Oops, I didn't realize Spiders 2.0 was also for newer versions, so there is hope for a compat with Better Animal Models.

**Comment by SandwichHorror** [comment]
Right, that's why I asked on this end instead of on Better Animal Models.
